src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,105 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  items: string[],
+  currentPage: number,
+  perPage: number,
+  onPageChange: (value: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  items, currentPage, perPage, onPageChange,
+}) => {
+  const lastPage = Math.ceil(items.length / perPage);
+
+  const displayPagination = Array.from({ length: lastPage }, (_, i) => i + 1)
+    .map(page => (
+      <li
+        key={page}
+        className={
+          cn(
+            'page-item',
+            { active: currentPage === page },
+          )
+        }
+      >
+        <a
+          href={`#${page}`}
+          data-cy="pageLink"
+          className="page-link"
+          onClick={() => onPageChange(page)}
```

Comment:
don't trigger `onPageChange` if a click was on the already selected item

Reply:
I got it. Ok :)